Formula/conftest.rb

```diff
@@ -0,0 +1,18 @@
+class Conftest < Formula
+  desc "Test your configuration files using Open Policy Agent"
+  homepage "https://www.conftest.dev/"
+  url "https://github.com/open-policy-agent/conftest/archive/v0.23.0.tar.gz"
+  sha256 "116bfd3c00fcaa31a3468cace6df5b03eeff0150d0d096aee53dd74c7c9647bd"
+  license "Apache-2.0"
+  head "https://github.com/open-policy-agent/conftest.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "-ldflags", "-X github.com/open-policy-agent/conftest/internal/commands.version=#{version}"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-X github.com/open-policy-agent/conftest/internal/commands.version=#{version}")
```

Reply:
Looks like you got rid of the commit that fixed this by accident. When you fix this, please squash all your commits together with the first one.

Reply:
Re-added and squashified.